<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAuActorRdfEditWidget</class>
 <widget class="QWidget" name="CAuActorRdfEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>319</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="buddy">
          <cstring>nameLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="nameLineEdit">
         <property name="text">
          <string/>
         </property>
         <property name="urlDropsEnabled">
          <bool>false</bool>
         </property>
         <property name="clickMessage">
          <string>Enter actor's name here</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="aliasesLabel">
         <property name="text">
          <string>Aliases:</string>
         </property>
         <property name="buddy">
          <cstring>aliasesLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="KLineEdit" name="aliasesLineEdit">
         <property name="text">
          <string/>
         </property>
         <property name="urlDropsEnabled">
          <bool>false</bool>
         </property>
         <property name="clickMessage">
          <string>Enter all actors aliases here, separating them with &quot;,&quot;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>Type:</string>
         </property>
         <property name="buddy">
          <cstring>typeComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="KComboBox" name="typeComboBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="roleLabel">
         <property name="text">
          <string>Role:</string>
         </property>
         <property name="buddy">
          <cstring>roleComboBox</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="KComboBox" name="roleComboBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="ageLabel">
         <property name="text">
          <string>Age:</string>
         </property>
         <property name="buddy">
          <cstring>ageSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="ageSpinBox">
         <property name="specialValueText">
          <string>Not specified</string>
         </property>
         <property name="maximum">
          <number>1000000000</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KLineEdit" name="shortDescrLineEdit">
         <property name="clickMessage">
          <string>One line description here to show in lists</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="shortDescrLabel">
         <property name="text">
          <string>Short description:</string>
         </property>
         <property name="buddy">
          <cstring>shortDescrLineEdit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="descrLabel">
       <property name="text">
        <string>Actor description:</string>
       </property>
       <property name="buddy">
        <cstring>descrRichTextWidget</cstring>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QToolButton" name="undoToolButton">
         <property name="text">
          <string>Undo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="redoToolButton">
         <property name="text">
          <string>Redo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="boldToolButton">
         <property name="text">
          <string>Bold</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="italicToolButton">
         <property name="text">
          <string>Italic</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="underlineToolButton">
         <property name="text">
          <string>Underline</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="leftAlignToolButton">
         <property name="text">
          <string>LeftAlign</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="centerAlignToolButton">
         <property name="text">
          <string>CenterAlign</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="rightAlignToolButton">
         <property name="text">
          <string>RightAlign</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="KRichTextWidget" name="descrRichTextWidget">
       <property name="clickMessage">
        <string>Enter actor's description here</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KRichTextEdit</class>
   <extends>KTextEdit</extends>
   <header>krichtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KRichTextWidget</class>
   <extends>KRichTextEdit</extends>
   <header>krichtextwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
